drivers/linkedin.go

```diff
@@ -1,39 +1,72 @@
 package drivers
 
 import (
-	"net/http"
+    "encoding/json"
+    "fmt"
+    "net/http"
+    "os"
 
-	"github.com/danilopolani/gocialite/structs"
-	"golang.org/x/oauth2/linkedin"
+    "github.com/danilopolani/gocialite/structs"
+    "golang.org/x/oauth2/linkedin"
 )
 
 const (
-	linkedinDriverName = "linkedin"
+    linkedinDriverName = "linkedin"
 )
 
 func init() {
-	registerDriver(linkedinDriverName, LinkedInDefaultScopes, LinkedInUserFn, linkedin.Endpoint, LinkedInAPIMap, LinkedInUserMap)
+    registerDriver(linkedinDriverName, LinkedInDefaultScopes, LinkedInUserFn, linkedin.Endpoint, LinkedInAPIMap, LinkedInUserMap)
 }
 
 // LinkedInUserMap is the map to create the User struct
 var LinkedInUserMap = map[string]string{
```

Comment:
We should provide the same output fields here or it would be a breaking change

Reply:
@danilopolani no changes to the fields it is just the padding.

Reply:
Yeah sorry Github put the comment on the wrong section, the comment was about the new `LinkedInUserMap`, it hasn't anymore the previous fields like `username`, `avatar` and so on but only `FirstName` and `LastName` (+ `Email` added later).

If we remove or change the output of those fields, an application that already uses them and updates the package will break for sure

Reply:
applied